contracts/TablelandTables.sol

```diff
@@ -66,8 +69,36 @@ contract TablelandTables is
     function runSQL(
         address caller,
         uint256 tableId,
-        string memory statement
+        string calldata statement
+    ) external payable override whenNotPaused nonReentrant {
+        _runSQL(caller, tableId, statement);
+    }
+
+    /**
+     * @dev See {ITablelandTables-runSQLs}.
+     */
+    function runSQLs(
+        address caller,
+        ITablelandTables.Runnable[] calldata runnables
     ) external payable override whenNotPaused nonReentrant {
+        if (runnables.length > RUNNABLES_MAX_LENGTH) {
+            revert MaxStatementCountExceeded(
+                runnables.length,
+                RUNNABLES_MAX_LENGTH
+            );
+        }
+
+        for (uint256 i = 0; i < runnables.length; i++) {
+            // simple pass along of each set of runSQL calls
+            _runSQL(caller, runnables[i].tableId, runnables[i].statement);
+        }
+    }
+
+    function _runSQL(
+        address caller,
+        uint256 tableId,
+        string calldata statement
+    ) private {
         if (
             !_exists(tableId) ||
             !(caller == _msgSenderERC721A() || owner() == _msgSenderERC721A())
```

Comment:
this (and others in this contract) set of conditional logic is being fully tested but the branch coverage report is saying that it's not.
I split this into logically identical but separate conditional statements and the branch coverage number will go up.  Not sure if this is a problem with hardhat coverage, or istanbul, or how we are using them, but I reduced the threshold from 96 to 93 to get tests passing.

Reply:
[update] Looks like I was reading the coverage report wrong.  The reduction in branch coverage was due to there being no tests for the `nonReentrant` function modifier.  I added some and now the only uncovered branch is in the`initializer` modifier, and I can't find a way to test that.